liquibase-core/src/main/java/liquibase/change/core/DropColumnChange.java

```diff
@@ -215,7 +215,7 @@ public boolean createThisIndex(Index index) {
             sqlStatements = new SqlStatement[statements.size()];
             return statements.toArray(sqlStatements);
         }
-        return sqlStatements;
+        return null;
```

Comment:
updating this to return an empty array as this is going to be use like this: `Arrays.asList(generateStatementsForSQLiteDatabase(database)`